### PR TITLE
Update functionality connected, read employees connected

### PR DESCRIPTION
**Pull request by sterlingwalker**
- Employee table now pulls from the server. (From now on to test with demo data you must change `isDemo`  to `true`)

- Employee update is fully functional

- Demo data file edited to match database

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
+import { employees } from './demo'
 const apiPath = '/hr-server/api/v1/';
 
+const isDemo = false //Change to true for demo data
```
Comment (yonimn2000) [inline src/api.js:4]: Could you please make sure the semicolon usage is consistent throughout this file?

**Review by yonimn2000 — CHANGES_REQUESTED** [review]

File: src/api.js (hunk 1 above)
Comment (sterlingwalker) [reply src/api.js]: Good point, I'm not used to adding semicolons for js. Npm has a package called `Prettier` that goes through and formats the code nicely. I'll add it in to this project in the next PR.

**Review by yonimn2000 — APPROVED** [review]
Cool, thanks.

**Comment by sterlingwalker** [comment]
@yonimn2000 So for this PR as well as the server I removed Dennes as a reviewer just to lighten his load a bit. I think these might be ready to go in.

**Pull request merged**